### DISCUSSION TITLE
Reg. enabling the forwarding from another BR to Thread network

### DISCUSSION THREAD

**Discussion by pragun01** [openthread/openthread#9346] [Q&A]
I am working on thread certification, in test 5.10.9 DUT as BBR receives ping request from another BR, DUT needs to forward ping request to thread network, but packet is dropped at DUT.
What are the routing configuration I need to modify for DUT to forward ping request?

**Comment by jwhui** [comment]
POSIX platform implementation should already manage routes for prefixes assigned to the Thread network:

https://github.com/openthread/openthread/blob/b03022b2afc52597f030379093a723fbf5f850c3/src/posix/platform/netif.cpp#L382

Can you check the unicast address and prefix length assigned to your Thread network interface?
